core/trino-main/src/main/java/io/trino/execution/scheduler/StageTaskSourceFactory.java

```diff
@@ -22,6 +22,7 @@
 import com.google.common.collect.ImmutableSet;
```

Comment:
Could you extend commit message to describe what this actually means?

Reply:
Changed. Hope it is better now.